spec/controllers/admin/users_controller_spec.rb

```diff
@@ -0,0 +1,235 @@
+require 'spec_helper'
+
+RSpec.describe Admin::UsersController do
+  describe "GET #index" do
+    before(:each) do
+      FactoryGirl.create_list(:user, 3)
+    end
+
+    context "when user is a client" do
+      let(:user) { FactoryGirl.create(:user) }
+      it "redirects to applicatioon root" do
+        sign_in user
```

Comment:
Możesz chyba to `sign_in user` wynieść do globalnego `before(:each)` - powinien inicjalizować Ci usera z `let`-ów wedle scope'u działania.